pkg/worker/worker_test.go

```diff
@@ -21,12 +21,16 @@ func (s *workerSuite) TestWorkerProcessorReturnsAnError() {
 	acker := &ackMock{}
 	publisher := make(chan mq.PublishMessage)
 	consumer := make(chan amqp091.Delivery)
-	incomingMessage := pipeline.Message{}
-	expectedMessage := workerError{
-		Worker:    "some-worker",
-		MessageID: "",
-		Topic:     "message came from this topic",
-		Error:     "unexpected error occurred!!",
+	incomingMessage := pipeline.Message{
+		ID: "blabla",
```

Comment:
Perhaps use uuid.NewString() w/ googles uuid package. Just to keep it consistent.